webpack.prod.config.ts

```diff
@@ -0,0 +1,80 @@
+import * as webpack from 'webpack';
+import * as CompressionWebpackPlugin from 'compression-webpack-plugin';
+import * as HTMLWebpackPlugin from 'html-webpack-plugin';
+
+const { merge } = require('webpack-merge');
+
+const common = require('./webpack.common.config');
+
+const packageJson: {
+  dependencies: { [libName: string]: string };
+  devDependencies: { [libName: string]: string };
+} = require(require.resolve('./package.json'));
+
+/** Get clean version of a version string of package.json entry for a package by
+ * extracting only alphanumerics, hyphen, and period. Note that this won't
+ * produce a valid URL for all possible NPM version strings, but should be fine
+ * on those that are absolute version references.
+ * Examples: '1', '1.0', '1.2.3', '1.2.3-alpha.0'
+ */
+function absoluteVersion(version: string) {
+  return version.replace(/[^\d.\-a-z]/g, '');
+}
+
+/** CDN path in case we would use minified react and react-DOM */
```

Comment:
nit: replace `in case we would use` with `to use ... instead of prebuild version` 
or something like that